specification/cognitiveservices/OpenAI.Inference/models/audio/commons.tsp

```diff
@@ -31,6 +31,17 @@ Requesting format 'json' will result on only the 'text' field being set.
 For more output data use 'verbose_json.
 """)
 model AudioTranscriptionOptionsCommon {
+    @doc("""
+    The content type for the operation. Always multipart/form-data for this operation.
+    You need to set your content separtor with the boundary value hardcode here: "azure_openai"
+    """)
+    @header("content-type")
+    contentType: "multipart/form-data; boundary=azure_openai";
+
+    @doc("The content length of the operation. This needs to be provided by the caller.")
+    @header("content-length")
+    contentLength: int64;
```

Comment:
This ends up emitting a bit strangely (at least in .NET) with the generated method signatures requiring `contentLength` as a parameter -- the actual content length is dependent on the serialization of the (peer) options instance as well as the internal details on how multipart is serialized, so I don't believe that parameter makes sense.

We should check with the TypeSpec experts on the intended way to make "must be internally populated headers" show up -- if the convention is that these get promoted into a method-level property, this seems odd.

Concretely, I'm seeing .NET emit signatures like this, and I don't know how a caller (internal or not) would ever be able to use the contentLength parameter:

```csharp
internal virtual async Task<Response<AudioTranscriptionSimpleJson>> GetAudioTranscriptionSimpleJsonAsync(
    string deploymentId,
    long contentLength,
    AudioTranscriptionOptionsSimpleJson audioTranscriptionOptionsSimpleJson,
    CancellationToken cancellationToken = default)
```

Reply:
The sames is true for Java, but for me this is a feature. I can then provide the `contentLength` at the protocol method level. I am expecting for all these methods to be private from the perspective of the user. We'd then have a single public entry point from the users perspective. 

Although, maybe I am not understanding your concern here. I agree, let's open this up to the meeting on Monday or Wednesday :)